Spigot-Server-Patches/0596-Added-firing-of-PlayerChangeBeaconEffectEvent.patch

```diff
@@ -0,0 +1,52 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Jake Potrebic <jake.m.potrebic@gmail.com>
+Date: Wed, 24 Jun 2020 15:14:51 -0600
+Subject: [PATCH] Added firing of PlayerChangeBeaconEffectEvent
+
+
+diff --git a/src/main/java/net/minecraft/server/ContainerBeacon.java b/src/main/java/net/minecraft/server/ContainerBeacon.java
+index 288390bf9210f0fcc7c85e3093a4924614dc1573..b4df9e9e4df6b7ca657d812fada1ddb2e654c668 100644
+--- a/src/main/java/net/minecraft/server/ContainerBeacon.java
++++ b/src/main/java/net/minecraft/server/ContainerBeacon.java
+@@ -1,6 +1,11 @@
+ package net.minecraft.server;
+ 
+ import org.bukkit.craftbukkit.inventory.CraftInventoryView; // CraftBukkit
++// Paper start
++import io.papermc.paper.event.player.PlayerChangeBeaconEffectEvent;
++import org.bukkit.entity.Player;
++import org.bukkit.potion.PotionEffectType;
++// Paper end
+ 
+ public class ContainerBeacon extends Container {
+ 
+@@ -128,9 +133,14 @@ public class ContainerBeacon extends Container {
+ 
+     public void c(int i, int j) {
+         if (this.d.hasItem()) {
+-            this.containerProperties.setProperty(1, i);
+-            this.containerProperties.setProperty(2, j);
+-            this.d.a(1);
++            // Paper start
++            PlayerChangeBeaconEffectEvent event = new PlayerChangeBeaconEffectEvent((Player) this.player.player.getBukkitEntity(), PotionEffectType.getById(i), PotionEffectType.getById(j), this.containerAccess.getLocation().getBlock());
++            if (event.callEvent()) {
++                this.containerProperties.setProperty(1, event.getPrimary() == null ? 0 : event.getPrimary().getId());
++                this.containerProperties.setProperty(2, event.getSecondary() == null ? 0 : event.getSecondary().getId());
++                this.d.a(1);
++            }
++            // Paper end
+         }
+ 
+     }
+diff --git a/src/main/java/net/minecraft/server/PlayerConnection.java b/src/main/java/net/minecraft/server/PlayerConnection.java
```

Comment:
Get rid of the patch for PlayerConnection class.

Reply:
ugh lol. I thought I got it, missed a newline I guess